inclusive-design/wecount.inclusivedesign.ca
# feat: remove "apply filters" button

## Description

Remove the "apply filters" button. Hitting Enter after entering a search term or selecting/unselecting a checkbox will trigger the apply action right away.

## Steps to test

1. Go to the resource page;
2. Enter a search term, hit Enter, the search should be performed;
3. Select a checkbox in any filter, the filtering happens right away;
4. Repeat step 2, 3 to verify the combination of search and filter.

**Expected behavior:** 
See above.

## Additional information



## Related issues
